python/grass/temporal/register.py

```diff
@@ -603,37 +603,34 @@ def register_map_object_list(
     import grass.pygrass.modules as pymod
     import copy
 
-    dbif, connection_state_changed = init_dbif(dbif)
+    dbif, connection_state_changed = init_dbif(None)
```

Comment:
At the previous place in these changes, a same line was changed from `init_dbif(None)` to `init_dbif(dbif)`. What's the reasoning?

Reply:
I find the TGIS database interactions somewhat confusing, so I am not 100% sure what is happening exactly. In this it was necessay to pass a database interface and avoid changing it. But that is more from test-driven development than from deep understanding of the inner database interactions in TGIS...